src/pages/Embark/index.tsx

```diff
@@ -325,6 +325,20 @@ export const EmbarkRoot: React.FunctionComponent<EmbarkRootProps> = (props) => {
                         storageState.session.setSession({
                           ...storageState.session.getSession(),
                           quoteIds: [store.quoteId],
+                          bundledQuoteIds: [],
+                        })
+                      }
+                      if (
+                        store.norwegianHomeContentsQuoteId &&
+                        store.norwegianTravelQuoteId
+                      ) {
+                        storageState.session.setSession({
+                          ...storageState.session.getSession(),
+                          quoteIds: [],
+                          bundledQuoteIds: [
+                            store.norwegianHomeContentsQuoteId,
+                            store.norwegianTravelQuoteId,
+                          ],
```

Comment:
unsure how to store these and how to add them to the session. This worked well but doesn't scale that well 😄

Reply:
Appending to the list may result in having too many quotes than are allowed in the bundle and the ids have to be unique...